common/djangoapps/student/tests/tests.py

```diff
@@ -724,9 +725,22 @@ def test_enrollment_multiple_classes(self):
         self.assertFalse(CourseEnrollment.is_enrolled(user, course_id2))
 
     @unittest.skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Test only valid in lms')
-    def test_activation(self):
+    @override_settings(LMS_SEGMENT_KEY="testkey")
```

Comment:
I'm wondering if the test failure has to do with using `override_settings` twice -- maybe try combining the two calls into one?